splits/README.md

```diff
@@ -0,0 +1,29 @@
+# Evaluation Setup of Natural Instruction V2
```

Comment:
Somewhere you may want to add a reference to your models (https://github.com/yizhongw/Tk-Instruct)